src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Resources/Theme/Dark.xaml

```diff
@@ -475,6 +479,9 @@
     <SolidColorBrush x:Key="FlyoutBackground" Color="{StaticResource AcrylicBackgroundFillColorDefault}" />
     <SolidColorBrush x:Key="FlyoutBorderBrush" Color="{StaticResource SurfaceStrokeColorFlyout}" />
 
+    <!--  GroupBox  -->
+    <SolidColorBrush x:Key="GroupBoxForeground" Color="{StaticResource SystemBaseHighColor}" />
```

Comment:
I'm confused why `SystemBaseHighColor` is used vs. `TextFillColorPrimaryBrush`? That should be the default color used everywhere for primary text:

![image](https://github.com/user-attachments/assets/f9f85461-f7fb-4b8b-9b7d-3a3aec73ceda)